ocrd_validators/ocrd_validators/workspace_validator.py

```diff
@@ -141,6 +147,20 @@ def _validate_dimension(self):
             if page.imageWidth != exif.width:
                 self.report.add_error("PAGE '%s': @imageWidth != image's actual width (%s != %s)" % (f.ID, page.imageWidth, exif.width))
 
+    def _validate_multipage(self):
+        """
+        Validate the number of images per file is 1 (TIFF allows multi-page images)
+
+        See `spec <https://ocr-d.github.io/mets#no-multi-page-images>`_.
+        """
+        for f in [f for f in self.mets.find_files() if f.mimetype.startswith('image/')]:
```

Comment:
```suggestion
        for f in self.mets.find_files():
            if not f.mimetype.startswith('image/'):
                continue
```

Reply:
What's the difference?

Reply:
None – that's the point! It's more readable and more Pythonic IMHO. But as it says, it's only a suggestion.

Reply:
`for f in [f for f in` is quirky I agree but we use that pattern consistently in the tests and I don't want to change them all. Plus for tests I often find one-liners helpful to make it easier to copy/paste/replace all.